src/frontend/src/common/components/library/data_subview/components/createTreeModal/style.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
I assume you're going to add some stuff in here immediately after merging this, so I don't mind it here. I'm wondering if, generally, we can avoid creating files that don't have anything in them? (Not sure whether linter requires a style sheet for every component or something).

Reply:
I was a little lax because we were adding this to the feature branch and I was going to start using this file right away so i left it in. I won't be this lazy in the next PR 😆